### PR TITLE
add http client

### PR DESCRIPTION
**Pull request by cowan-macady**
- add http client
- removed authorization from tests because the BE is broken for now
- (release will not be done until the authorization be is finished)

### CODE REVIEW

File: indykite_sdk/api.py [hunk 1]
```diff
@@ -1928,6 +1936,40 @@ def main():
         else:
             print("Invalid invitation response")
 
+    elif command == "get_http_client":
+        token = None
+
+        # generate an authenticated http client and generate a bearer token from the provided credentials
+        client_http = HttpClient()
+        response_http = client_http.get_http_client(token)
+        print(response_http.token_source.token.access_token)
```
Comment (kilimandzsaro) [inline indykite_sdk/api.py:1945]: Do we still need this print?

File: indykite_sdk/api.py [hunk 2]
```diff
@@ -1928,6 +1936,40 @@ def main():
...
+        credentials = client_http.get_credentials()
+        # call get_refreshable_token_source again to check the token is the same
+        response_source = client_http.get_refreshable_token_source(response_http.token_source, credentials)
+        print(response_source.token.access_token)
```
Comment (kilimandzsaro) [inline indykite_sdk/api.py:1949]: Do we still need this print?

File: indykite_sdk/api.py [hunk 3]
```diff
@@ -1928,6 +1936,40 @@ def main():
...
+        # call get_http_client again to generate another http client and check if the token is the same
+        response_http2 = client_http.get_http_client(response_http.token_source)
+        access_token = response_http2.get_token()
+        print(access_token)
```
Comment (kilimandzsaro) [inline indykite_sdk/api.py:1953]: Do we still need this print?

File: indykite_sdk/api.py [hunk 4]
```diff
@@ -1928,6 +1936,40 @@ def main():
...
+        # make a Knowledge API query
+        endpoint = args.base_url
+        end1 = os.getenv('INDYKITE_SDK_URL')
+        end2 = "gid:AAAAAi7tSAs-qkg_his0YnvKuJ4"
```
Comment (kilimandzsaro) [inline indykite_sdk/api.py:1957]: Is it ok that this value is hard coded?

File: indykite_sdk/api.py [hunk 5]
```diff
@@ -1928,6 +1936,40 @@ def main():
...
+        response_post = requests.post(endpoint, json=data, headers=headers)
+        print(response_post.text)
+
+    elif command == "get_refreshable_token_source":
+        token_source = None
+        client_http = HttpClient()
+        credentials = client_http.get_credentials()
+        response = client_http.get_refreshable_token_source(token_source, credentials)
+        access_bytes = response.token.access_token
+        print(access_bytes.decode('utf-8'))
```
Comment (kilimandzsaro) [inline indykite_sdk/api.py:1971]: Do we still need this print?

**Review by kilimandzsaro — CHANGES_REQUESTED** [review]

File: indykite_sdk/api.py (hunk 1 above)
Comment (cowan-macady) [reply indykite_sdk/api.py]: removed

File: indykite_sdk/api.py (hunk 3 above)
Comment (cowan-macady) [reply indykite_sdk/api.py]: removed

**Review by kilimandzsaro — APPROVED** [review]

**Pull request merged**